include/ts/apidefs.h.in

```diff
@@ -745,12 +745,7 @@ char const TS_VERSION_STRING[] = "@TS_VERSION_STRING@";
   };
 
   /* The values of this enum must match enum RecUpdateT in I_RecDefs.h */
-  enum TSRecordUpdateType {
-    TS_RECORDUPDATE_NULL,
-    TS_RECORDUPDATE_DYNAMIC,
-    TS_RECORDUPDATE_RESTART_TS,
-    TS_RECORDUPDATE_RESTART_TM,
-  };
+  enum TSRecordUpdateType { TS_RECORDUPDATE_NULL, TS_RECORDUPDATE_DYNAMIC, TS_RECORDUPDATE_RESTART_TS };
```

Comment:
Was this done by clang-format? Retaining the comma after the last item may keep this in a good shape.

Reply:
yes, added `,` after last item and now is back to normal.
Thanks.